Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -12,28 +12,32 @@ public enum ActorStates
 
 public class ActorPhysics : ActorComponent
 {
+	private new Transform transform;
```

Comment:
The idea is that the interface is the same but with only one GetComponent() call.